src/pages/index.js

```diff
@@ -75,9 +179,40 @@ cardAddButton.addEventListener("click", () => {
   addCardPopupForm.open();
 });
 
+avatarEditButton.addEventListener("click", () => {
+  avatarPopupForm.open();
+});
+
+//Class calls
 addFormValidator.enableValidation();
 editFormValidator.enableValidation();
 addCardPopupForm.setEventListeners();
 profilePopupForm.setEventListeners();
 cardImagePopup.setEventListeners();
-layerSection.renderItems();
+deletePopup.setEventListeners();
+avatarPopupForm.setEventListeners();
+avatarFormValidator.enableValidation();
+
+//API Calls
+api
+  .getUser()
+  .then((result) => {
+    userInfo.setUserInfo({
+      name: result.name,
+      description: result.about,
+      avatar: result.avatar,
+    });
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+api
+  .getInitialCards()
+  .then((res) => {
+    layerSection.setItems(res);
+    layerSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors

Reply:
Fixed.